fuzzers/introspector_driven_focus/patch.diff

```diff
@@ -0,0 +1,100 @@
+diff --git a/compiler-rt/lib/fuzzer/FuzzerFork.cpp b/compiler-rt/lib/fuzzer/FuzzerFork.cpp
```

Comment:
This patch isn't used by libFuzzer. I don't htink you need it.

Reply:
Thanks.